lib/hawkular/metrics/metric_api.rb

```diff
@@ -79,14 +79,15 @@ def query(tags = nil)
       # @param id [String]
       # @return [MetricDefinition]
       def get(id)
-        the_id = @client.hawk_escape id
```

Comment:
an alternative way to write this would be adding in the head of the file
`require "erb"`
`include ERB::Util`

and then just call in the code `url_encode` 
it does exactly the same thing, but just is shorter